pylatex/utils.py

```diff
@@ -249,6 +249,25 @@ def bold(s, *, escape=True):
     return NoEscape(r'\textbf{' + s + '}')
 
 
+def text_color(s, color, *, escape=True):
+    r"""Change the string color.
+
+    Args
+    ----
+    s : str
+        The string to be formatted
+    escape : bool
+        If true the text will be escaped
+    color : str
+        The name of the color for the text
+    """
+
+    if escape:
+        s = escape_latex(s)
+
+    return NoEscape(r"\textcolor{" + color + "}{" + s + "}")
+
+
 def italic(s, *, escape=True):
     r"""Make a string appear italicized in LaTeX formatting.
```

Comment:
Just leave italic and the other original ones here for now. After this is merged I'll just add them to the basic module as well.